subsys/bluetooth/host/l2cap.c

```diff
@@ -2566,6 +2610,101 @@ int bt_l2cap_ecred_chan_connect(struct bt_conn *conn,
 
 	return err;
 }
+
+#if defined(CONFIG_BT_TESTING)
+static struct bt_l2cap_le_chan *l2cap_find_pending_reconf(struct bt_conn *conn)
+{
+	struct bt_l2cap_chan *chan;
+
+	SYS_SLIST_FOR_EACH_CONTAINER(&conn->channels, chan, node) {
+		if (BT_L2CAP_LE_CHAN(chan)->pending_rx_mtu) {
+			return BT_L2CAP_LE_CHAN(chan);
+		}
+	}
+
+	return NULL;
+}
+
+int bt_l2cap_chan_reconfigure(struct bt_l2cap_chan **chan, uint16_t mtu)
+{
+	struct bt_l2cap_ecred_reconf_req *req;
+	struct bt_conn *conn = NULL;
+	struct bt_l2cap_le_chan *ch;
+	struct net_buf *buf;
+	uint8_t ident;
+	int i;
+
```

Comment:
Missing NULL check for `chan` (in case that the user supplies NULL to the first parameter

Reply:
There is `if (!chans) {` a few lines below.